### PR TITLE
doc: doc updates related to new SES

### PR DESCRIPTION
**Pull request by b-gent**
New image and updated description.

Signed-off-by: Bartosz Gentkowski <bartosz.gentkowski@nordicsemi.no>

### CODE REVIEW

File: doc/nrf/gs_programming.rst [hunk 1]
```diff
@@ -91,7 +91,7 @@ You can download it from the following links:
       a. Select your project in the Project Explorer.
```
Comment (b-gent) [inline doc/nrf/gs_programming.rst]: @tejlmand  I know that this section probably needs some more changes but I wasn't sure which steps exactly are affected. Can you help me by leaving comments?

**Comment by b-gent** [comment]
@tejlmand I know that the RST file probably needs some more changes but I wasn't sure which steps exactly are affected. Can you help me by leaving comments?

File: doc/nrf/gs_programming.rst [hunk 2]
```diff
@@ -91,7 +91,7 @@ You can download it from the following links:
...
       #. From the menu, select **Build -> Build Solution**.
       #. When the multi-image build completes, you can flash the sample to a connected board.
-         To do this, select **Target -> Download File -> Download Intel Hex File**.
+         To do this, select **Target -> Download zephyr/merged.hex**.
          Navigate to the ``zephyr`` folder in your build directory and choose ``merged.hex``.
```
Comment (tejlmand) [inline doc/nrf/gs_programming.rst]: I would rephrase the text from line: 77-96: into:
```
 To build and flash an application:
      a. Select your project in the Project Explorer.
      #. From the menu, select **Build -> Build Solution**.
      #. When the build completes, you can flash the sample to a connected board.
         To do this, select
              **Target -> Download zephyr/zephyr.elf**. (single image application)
              **Target -> Download zephyr/merged.hex**. (multi image application)

      .. note::
	   Alternatively, choose the **Build and Debug** option.
	   **Build and Debug** will build the application and flash it when
the build completes.
```

Comment (b-gent) [reply doc/nrf/gs_programming.rst]: thanks @tejlmand  , please verify and approve

Just to double-check  - the new SES version is 4.18?
